src/app/components/image/image.css

```diff
@@ -0,0 +1,16 @@
+.image--round {
+  border-radius: 50%;
+  width: 15rem;
```

Comment:
Look at this , please use correct naming for the classes ( esp. if you are using BEM Model )